src/extension/update.spec.ts

```diff
@@ -83,7 +83,7 @@ const makeStubs = () => ({
             executeCommand: fake() // (command: string, ...rest: any[]) => undefined
         },
         extensions: {
-            getExtension: () => ({ packageJSON: { version: '3.0.0' } })
+            getExtension: () => ({ packageJSON: { version: '3.0.1' } })
```

Comment:
>1 [](start = 64, length = 1)

I don't know the right way to increment the version.

Reply:
This is a unit test for the auto-update feature. It's mock data and does not need to be updated.

The version source of truth is in `package.json`.

Reply:
Thanks!

---
In reply to: [509609462](https://github.com/microsoft/sarif-vscode-extension/pull/364#discussion_r509609462) [](ancestors = 509609462)